cmd/util_test.go

```diff
@@ -408,31 +408,27 @@ func TestGetConfigFilePath(t *testing.T) {
 		t.Fatalf("Failed to create temp directory")
 	}
 	defer os.RemoveAll(cfgDir)
-	d := filepath.Join(cfgDir, "1.8")
+	d := filepath.Join(cfgDir, "cis-1.4.1")
 	err = os.Mkdir(d, 0666)
 	if err != nil {
 		t.Fatalf("Failed to create temp file")
 	}
 	ioutil.WriteFile(filepath.Join(d, "master.yaml"), []byte("hello world"), 0666)
 
 	cases := []struct {
-		specifiedVersion string
-		runningVersion   string
+		benchmarkVersion string
+		specifiedVersion bool
 		succeed          bool
 		exp              string
 	}{
-		{runningVersion: "1.8", succeed: true, exp: d},
-		{runningVersion: "1.9", succeed: true, exp: d},
-		{runningVersion: "1.10", succeed: true, exp: d},
-		{runningVersion: "1.1", succeed: false},
-		{specifiedVersion: "1.8", succeed: true, exp: d},
-		{specifiedVersion: "1.9", succeed: false},
-		{specifiedVersion: "1.10", succeed: false},
+		{benchmarkVersion: "cis-1.4.1", specifiedVersion: true, succeed: true, exp: d},
+		{benchmarkVersion: "cis-1.5.0", specifiedVersion: false, succeed: true, exp: d},
+		{benchmarkVersion: "1.1", succeed: false},
```

Comment:
I think we should retain the tests that cover the user specifying the kubeVersion. Not sure we haven't also lost cases for when the user doesn't specify the version (no benchmark version and no kube version) and we have to work it out by getting the running version?

Reply:
💯